### DISCUSSION TITLE
请各位理性沟通, 合理沟通, 友善沟通! 有话好好说, 我们会尽力帮你们解决问题的!

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#836] [Announcements]
如果你們要我們幫你們解決問題那就請各位好好的溝通. 不要動不動就說一些不好的話. 我們正當的做生意. 各位都是成年人了, 請注意你們所說的話. 不要動不動就要吵架和詆毀! 谢谢各位的合作!

**Comment by ycjdixon** [comment]
首先要把产品做好，把服务做好。近一年的时间了，正常使用的时间屈指可数，用户的体验确实不好。要理解大家的心情。

**Comment by Lewis0000221** [comment]
体验一天比一天差，开通不够一周，今天手机的专业版还连不上了，前几天用wifi一直连不上，要用手机流量才能连，现在手机流量都连不上了

**Comment by mcutdgreat** [comment]
请问我的退款流程还没有走完吗？已经一个月了

**Comment by ruochen800** [comment]


**Reply by MaxMKMax** [reply]
Hi @ruochen800 I thought your issues has been solved well previously? May i have again your account email and payment receipt? So i can resubmit the issues to the payment team and help you solve the issues. Thanks.

**Comment by yjiadi** [comment]
@Mike777Khoo ，您好！
上个月我才续费升级，因为重装系统，我再上蓝灯后，邮箱收不到的验证码。
我的邮箱是yjiadi@sina.cn，但不知道什么时候账号变成yiadi@sina.cn了。
麻烦您帮我看下怎么回事，谢谢！

**Comment by YG1017** [comment]
你们的客服内部邮件 最近快一年多的时间 一直在不断得给我的个人邮箱发送 内部邮件
能否尽快处理掉，不要骚扰我的邮箱了

**Reply by YG1017** [reply]
我的邮箱  179942441@qq.com

**Comment by XINYUAN1986** [comment]
专业版账户无法登录，能接收到授权码，输入授权码显示授权成功，点击下一步跳转回主页后，依然显示是免费版。账户1017art@gmail.com

**Comment by MaxMKMax** [comment]
@ ALL 各位. 谢谢你们的反馈. 我会帮你们提交问题和反馈上去的. 谢谢你们的理解和支持蓝灯.

**Comment by wlln157** [comment]
蓝灯升级专业版付费方式只有银行卡，我需要一个微信或支付宝的方式，为这个账号升级专业版“[55385338@qq.com](mailto:55385338@qq.com)”
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@wlln157 您好. 我让支付团队给您发支付链接. 谢谢您支持蓝灯!

**Reply by MaxMKMax** [reply]
@wlln157 
我们先给您免费用Pro. 等支付团队弄链接给您.
623GH-9JMYC-4Y8R7-BKF7Q-XRDD8

**Comment by wlln157** [comment]
***@***.***”，给我一个付费链接。着急使用蓝灯，谢谢！



--------------------------------

 

 

----- 原始邮件 -----
发件人：ruochen800 ***@***.***>
收件人：getlantern/lantern-forum-cn ***@***.***>
抄送人：wlln157 ***@***.***>,  Comment ***@***.***>
主题：Re:_[getlantern/lantern-forum-cn]_请各位理性沟通,_合理沟通,_友善沟通!_有话好好说,_我们会尽力帮你们解决问题的!_(Discussion_#836)
日期：2024年09月13日 12点22分


You've tried to  resolved the issue, but the time was not fully compensated. I was given four activation codes, but only two worked: one for 20 days and the other for 30 days, adding up to 50 days. The two-month activation codes couldn't be used, so there is still a shortfall of 60 days.

content-2094869512.jpg (view on web)

1.png (view on web)

2.png (view on web)

**Comment by slover1965** [comment]
一直在好好沟通，问题是ios端一点没给做啊，到现在连个测试版都发不出来

**Comment by Honeydukes3** [comment]
你好！我用支付宝购买蓝灯专业版后，为什么无法升级使用？
账号是771324905donald@tutanota.com
@Mike777Khoo

**Reply by Honeydukes3** [reply]
就在昨天

**Reply by MaxMKMax** [reply]
@Honeydukes3 您好. 已经帮您提交问题上去了. 您也给我提供支付单据吧. 谢谢您!

**Comment by dongc** [comment]
专业版账户无法登录，邮箱能接收到授权码，输入授权码显示授权成功，点击下一步跳转回主页后，依然显示是免费版。
账户[dongc2010@gmail.com](mailto:dongc2010@gmail.com) 有效到 2026 年 2 月 @Mike777Khoo

**Reply by MaxMKMax** [reply]
@dongc 我帮您提交问题上去. 请您也提供我支付单据. 谢谢!

**Comment by juzhier** [comment]
问题是你们服务真的很差，我前几次都是在你们第三方代理商付的款，有什么问题，发他们的联系邮箱秒回复，都可以及时处理。这次在你们主页在线支付，收不到验证码，已经在论坛反映两次，让我发支付信息，我跟帖发了，让我发邮箱，我也邮箱联系了。结果再没人理我。从8月8日付款到现在，我一直无法使用蓝灯专业版，而你们经常第一次回复完就消失，没有专人跟进处理，售后如此差，叫人怎么相信你们？？

**Reply by MaxMKMax** [reply]
@juzhier 您好. 已经在您的帖子上回复您了. 也帮您提交问题上去了.

**Comment by yjiadi** [comment]
@***@***.******@***.***了。麻烦您帮我看下怎么回事，谢谢！Message ID: ***@***.***>

**Comment by htc4wyz** [comment]
我的IOS设备下载登录后，VPN连接中，连接不上

**Comment by slover1965** [comment]
你们现在续费也是一坨，以前支付宝就可以，现在只能银行卡付费？支付宝怎么也不支持了，想续费都续不了。
